fe/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java

```diff
@@ -85,6 +90,71 @@ public MaterializedViewHandler() {
         super("materialized view");
     }
 
+
+    // for batch submit rollup job, tableId -> jobId
+    // keep table's not final state job size. The job size determine's table's state, = 0 means table is normal,otherwrise is rollup
+    private Map<Long, Set<Long>> tableNotFinalStateJobMap = new ConcurrentHashMap<>();
+    // keep table's running job,used for concurrency limit
+    private Map<Long, Set<Long>> tableRunningJobMap = new ConcurrentHashMap<>();
+    // make sure that job's dependency just init once
+    private volatile boolean hasBuildJobDependency = false;
```

Comment:
For simple. I think we needn't introduce the `Dependency` concept. We could check the
`alter statement` and require the from index must exist firstly, and then, we needn't consider the rollup job dependency.

Reply:
Good suggestion;It seems check whether baseRollup can be found in catalog is enough.

Reply:
No,Dependency is necessary;
Because int both parent job is cancel or parent is not final case, we all can't find parent index in catalog.Only 'parent index can be seen in catalog' can't decide whether current rollup job is  ready to run